owncloud/phoenix
# Feature/private link on external storage

## Description
This PR holds two changes:
1. In case of an mounted external storage the privateLink holdsing the fileid is useless. Each user will have a different fileid. In this case we use the current browser url as private link. Not really private anymore - but it works

2. In case any url to phoenix is used in an unauthenticated browser the route will be stored in the session storage of the browser and will be used as soon as the login process is finished, This replaces the private link specific storage.

## How Has This Been Tested?
- mount an external storage as admin to the connected owncloud instance
- login into phoenix and create a private link to a file within that mounted external storage
- the link will point to a full phoenix url
- open a new browser where no user is logged in
- paste the url into the browser
- see the login screen
- login
- user will be redirected to the linked file/folder in the external storage

## Screenshots (if appropriate):

## Types of changes


## Checklist:



## Open tasks:
